src/main/resources/paas.yaml

```diff
@@ -26,8 +26,8 @@ externalConfigDirectory: /tmp
 
 downloadConfigs: true
 
-fieldsYamlLocation: classpath://config/fields.yaml
-registersYamlLocation: classpath://config/registers.yaml
+fieldsYamlLocation: https://field.discovery.openregister.org/records.yaml
```

Comment:
Or just make it possible to set anything. Then you could have your own register/field registers deployed on PaaS.